wazo_calld/auth.py

```diff
@@ -18,15 +18,13 @@
 Unauthorized = auth_verifier.Unauthorized
 
 
-def get_token_user_uuid_from_request(auth_client, token=None):
-    if not token:
-        token = request.headers.get('X-Auth-Token') or request.args.get('token')
```

Comment:
This is not the default behavior of `AuthVerifier`: it will only look at the header, not the query string.
`AuthVerifier` needs to be instanciated with `extract_token_id = extract_token_id_from_query_or_header`. This will introduce a slight change in behavior: the token from query string will take precedence over the header.

Reply:
Actually, the only place needing the `token=...` query string is voicemail recordings. Those places already have specific token extraction logic:
```python
@required_acl(
        'calld.voicemails.{voicemail_id}.messages.{message_id}.recording.read',
        extract_token_id=extract_token_id_from_query_or_header,
    )
```